File: app/src/main/java/org/michaelbel/moviemade/ui/modules/trailers/TrailersMvp.java
package org.michaelbel.moviemade.ui.modules.trailers;

import com.arellomobile.mvp.MvpView;

import org.michaelbel.moviemade.annotation.EmptyViewMode;
import org.michaelbel.moviemade.data.dao.Video;

import java.util.ArrayList;
import java.util.List;

public interface TrailersMvp extends MvpView {

    void setTrailers(List<Video> trailers);

    void setError(@EmptyViewMode int mode);
}

File: app/src/main/java/org/michaelbel/moviemade/ui/modules/trailers/TrailersMvp.kt
package org.michaelbel.moviemade.ui.modules.trailers

import com.arellomobile.mvp.MvpView
import org.michaelbel.moviemade.annotation.EmptyViewMode
import org.michaelbel.moviemade.data.dao.Video

interface TrailersMvp : MvpView {

    fun setTrailers(trailers: List<Video>)

    fun setError(@EmptyViewMode mode: Int)
}